tools/sgcollect_info

```diff
@@ -454,6 +454,11 @@ def main():
 
     # Collect the SG binary
     sg_binary_path, _ = get_paths_from_expvars(sg_url)
+    if options.sync_gateway_executable_path is not None and len(options.sync_gateway_executable_path) > 0:
+        if not os.path.exists(sync_gateway_executable_path):
```

Comment:
Reposting as I put the comment in the wrong place.

Well we already log if we failed to collect a binary (and the path that we tried to fetch it from), either from the exp_var binary path or the user supplied one (https://github.com/couchbase/sync_gateway/blob/master/tools/sgcollect_info#L461). 
I suppose I can add a more specific one saying that we couldn't find a binary path from exp_vars (if that's possible) and there was no user supplied path either.

Is that the kind of thing you were looking for?

If not please could you clarify what you're asking for logging-wise?